c2corg_ui/templates/image/view.html

```diff
@@ -0,0 +1,161 @@
+<%!
+from c2corg_common.attributes import activities, image_categories, image_types
+from c2corg_ui.templates.utils import get_lang_lists
+from json import dumps
+%>
+
+<%inherit file="../base.html"/>
+<%namespace file="../helpers/common.html" import="show_title"/>
+
+<%namespace file="helpers/detailed_images_attributes.html" import="get_image_general, get_image_camera_settings"/>
+
+<%namespace file="../helpers/view.html" import="get_image_gallery, photoswipe_gallery,
+    show_attr, show_missing_langs_links, show_other_langs_links,
+    show_archive_data, show_route_title, get_route_activities, show_areas, show_float_buttons,
+    show_associated_waypoints, show_associated_routes, delete_association_confirmation_modal"/>
+
+<%
+image_id = image['document_id']
+image['doctype'] = 'images'
+other_langs, missing_langs = get_lang_lists(image, lang)
+%>
+
+<%block name="pagelang">lang="${lang}"</%block>
+<%block name="pagetitle"><title>${show_route_title(locale, True)}</title></%block>
+<%block name="metarobots">
+  % if version:
+    <meta name="robots" content="noindex,follow">
+  % else:
+    <meta name="robots" content="index,follow">
+  % endif
+</%block>
+
+<%block name="moduleConstantsValues">
+  module.value('mapFeatureCollection', {
+    "type": "FeatureCollection",
+    "properties": {},
+    "features": [{
+      "type": "Feature",
+      "geometry": {"type": "Point", "coordinates": [901624.4172679152, 5858812.695644485]},
+      "properties": {
+        "title": ${dumps(locale['title']) | n},
+        "lang": "${locale['lang']}",
+        "documentId": ${image['document_id']},
+        "module": "images"
+      }
+    }]
+  });
+
+  module.value('documentData', {
+    "document_id": ${image.get('document_id')},
+    "lang": "${lang}",
+    "type": "i"
+    % if not version:
+       , "associations": ${dumps(image.get('associations')) | n}
+    % endif
+  });
+</%block>
+
+<header class="view-details-title">
+  <h1 class="routes">
+    <div class="icons-header">${get_route_activities(image, 'top')}</div>
+    <span class="title">${show_route_title(locale)}</span><br>
```

Comment:
that's probably what's happening, we created the functions when routes arrived but then it turned out to be useful somewhere else than routes. I would prefer to  do it in a separate PR because it requires to change names in lots of pages and files.